src/components/Checkbox/README.md

```diff
@@ -25,5 +26,9 @@ const changeSelected = (item) => (value) => {
   <Checkbox disabled>
     Disabled Checkbox
   </Checkbox>
+  
+  <Checkbox onChange={changeSelected('fourth')} checked={state.fourth} customTick="orange">
```

Comment:
it's not clear that `customTick` should be a color in my opinion, what about `tickColor`? Also, we could have a regex for color validation, although maybe it's not really necessary.